src/config/dynamic-theme-fixes.config

```diff
@@ -19565,6 +19624,13 @@ CSS
 
 ================================
 
+my.babbel.com
+
+IGNORE INLINE STYLE
+#radix-\:r2\: > svg:nth-child(1) *
```

Comment:
Can you share some screens which flags exactly you include here?
I can not find it on main page and 2 flags I see on the main page are not touched with this selector:
![obraz](https://github.com/user-attachments/assets/773f1fb8-159b-4fd8-beca-fb3af6f81bc0)